pkgs/development/python-modules/dkimpy/default.nix

```diff
@@ -1,38 +1,28 @@
 { stdenv, fetchurl, openssl, makeWrapper, buildPythonApplication
-, pytest, dnspython }:
+, pytest, dnspython, pynacl, authres, python }:
 
 let
-  majorversion = "0.6";
-  minorversion = "2";
+  majorversion = "0.8";
+  minorversion = "1";
 in buildPythonApplication rec {
   pname = "dkimpy";
   version = "${majorversion}.${minorversion}";
 
   src = fetchurl {
     url = "https://launchpad.net/${pname}/${majorversion}/${majorversion}.${minorversion}/+download/${pname}-${version}.tar.gz";
-    sha256 = "1hagz8qk0v4ijfbcdq4z28bpgr2mkpr498z76i1vam2d50chmakl";
+    sha256 = "125xakqg2j5jp4k5kafpnpazh9d97ysgayrsgpvm3nkdj4g8hw1j";
   };
 
   checkInputs = [ pytest ];
-  propagatedBuildInputs =  [ openssl dnspython ];
+  propagatedBuildInputs =  [ openssl dnspython pynacl authres ];
 
   patchPhase = ''
-    substituteInPlace dknewkey.py --replace \
+    substituteInPlace dkim/dknewkey.py --replace \
       /usr/bin/openssl ${openssl}/bin/openssl
   '';
 
   checkPhase = ''
-    python ./test.py
-  '';
-
-  postInstall = ''
-    mkdir -p $out/bin $out/libexec
-    mv $out/bin/*.py $out/libexec
-    makeWrapper "$out/libexec/dkimverify.py" $out/bin/dkimverify
-    makeWrapper "$out/libexec/dkimsign.py" $out/bin/dkimsign
-    makeWrapper "$out/libexec/arcverify.py" $out/bin/arcverify
-    makeWrapper "$out/libexec/arcsign.py" $out/bin/arcsign
-    makeWrapper "$out/libexec/dknewkey.py" $out/bin/dknewkey
```

Comment:
Don;t you want to rename the executables any more?

Reply:
Upstream is doing that for me...

Reply:
Perfect.